helm/cluster-operator-chart/templates/configmap.yaml

```diff
@@ -23,6 +23,8 @@ data:
           certificate:
             ttl: '{{ .Values.Installation.V1.Auth.Vault.Certificate.TTL }}'
     service:
+      cluster:
+        address: 'http://cluster-service:8000'
```

Comment:
Somehow my flag lookup returns empty string for this. Debugging...

Reply:
Why do we even need that? Whatever is reconciled has the version bundle version we are interested in already. What do we need to ask `cluster-service` for?

Reply:
We have release version, but we don't have version bundles and we need to fetch them.

Reply:
Hmm... Now that I was actually taking a look, we need to change our provider extension. It does indeed have version bundle there, but that doesn't work for `cluster-operator` in any way. We should probably change it to be release version + version bundles or at least release version.

https://github.com/giantswarm/apiextensions/blob/master/pkg/apis/cluster/v1alpha1/aws_cluster_spec_types.go#L67

vs.

https://github.com/giantswarm/apiextensions/blob/master/pkg/apis/core/v1alpha1/cluster_guest_types.go#L14